docs/contributing.rst

```diff
@@ -237,10 +192,23 @@ Please before making a Pull Request about documentation changes run:
    $ make doc
 
 Once it finishes it will output the index html page
+
 ``open file:///.../aiohttp/docs/_build/html/index.html``.
 
 Go to the link and make sure your doc changes looks good.
 
+Note that this page doesn't automatically refresh itself. So, if you make more changes, build the docs again to see how they look on the page.
+
+.. note::
+   If you are on MacOS, you might need to install `graphviz <https://graphviz.org/>`_ first.
```

Comment:
This isn't limited to MacOS, the PR added an apt package to make this work in the CI:
https://github.com/aio-libs/aiohttp/pull/9359/files#diff-cde814ef2f549dc093f5b8fc533b7e8f47e7b32a8081e0760e57d5c25a1139d9R17-R18

So, maybe something like:
```
To build the docs you'll also need to install graphviz with your package manager (e.g. `apt install graphviz` or `brew install graphviz`).
```

Reply:
As this is prerequisite, I'd move this step above the `make doc` command, so users see it first.